Makefile

```diff
@@ -373,3 +373,13 @@ release-push-%:
 	$(MAKE) clean-bin
 	$(MAKE) ARCH=$* build
 	$(MAKE) ARCH=$* push
+
+# SvCat Kubectl plugin stuff
+############################
```

Comment:
Will you add this to the travis build as well? We should apso ad this to the docker image for catalog. No need to do the image stuff in this PR, but we should build it in travis from this PR.

Reply:
Fixed. Note that you have to run `./bin/svcat install` before it will show up as a kubectl plugin.